Tests/InterestNameValidationTests.swift

```diff
@@ -39,8 +39,8 @@ class InterestNameValidationTests: XCTestCase {
     }
 
     func testInterestsNameIsValid() {
-        let interests = ["hello"]
-        XCTAssertNoThrow(try PushNotifications.shared.setSubscriptions(interests: interests))
+        XCTAssertNoThrow(try PushNotifications.shared.setSubscriptions(interests: ["hello"]))
+        XCTAssertNoThrow(try PushNotifications.shared.setSubscriptions(interests: ["hello-world", "donuts", "pizza"]))
     }
 
     func testSomeInterestNamesAreInvalid() {
```

Comment:
In here, can you add an example for `hello|world`?

Reply:
Updated the test [here](https://github.com/pusher/push-notifications-swift/pull/48/commits/c2bc7e6f48240ab6ead9743e216de5383ac24e81).

`hello|world` is treated as invalid interest name.